lib/iex/lib/iex/introspection.ex

```diff
@@ -115,13 +126,37 @@ defmodule IEx.Introspection do
         end
         :ok
       else
-        :not_found
+        if has_callback?(mod, fun, arity) do
+          :behaviour_found
+        else
+          :not_found
+        end
```

Comment:
Maybe you could refactor this chained `if`s to a `cond`, or even better, refactor it to multiple functions with pattern matching?